tests/config/example_config.yml

```diff
@@ -0,0 +1,22 @@
+# DUT type should be either "local" or "ssh"
+# if ssh is selected, then ip, user and password fields are required
```

Comment:
Can we support ssh key auth?

Reply:
To do in the future if not possible in test-framework